modules/widget-visibility/widget-conditions.php

```diff
@@ -22,6 +22,12 @@ public static function init() {
 	}
 
 	public static function widget_admin_setup() {
+		$current_screen = get_current_screen();
+		// TODO: Replace `$current_screen->is_block_editor()` with `wp_should_load_block_editor_scripts_and_styles()` that is introduced in WP 5.6
```

Comment:
Can we just do a `function_exists()` or `is_callable()` check instead of having a todo comment? Or is someone committing to address this later?

Or is there concern that `wp_should_load_block_editor_scripts_and_styles()` isn't actually going to be added in WP 5.6 after all?

Reply:
We usually keep those `TODO` items, alongside with a specific WordPress version when a replacement function gets added. This way, when Jetpack will start requiring WordPress 5.6+, we'll be able to go back and edit the codebase around all `TODO` comments that refer to that in the codebase.

You're right though, we could start using the function right now (it's been added already, see [related ticket](https://core.trac.wordpress.org/ticket/51330)), with a `function_exists()` check, this way this would make our job easier and faster whenever we have to come back when Jetpack will require 5.6+. :)

@tyxla What do you think?

Reply:
Hey, folks! 

First of all, I did consider using the function directly, with a `function_exists` as you suggested. And if this was an instance where we don't have any alternative option, I likely would have used it that way.

However, the mechanism we're using here is totally safe and works in any modern WordPress. I'd say it's more stable if we keep it that way right now.

My rationale is that although the function is merged, it exists only in a development version of WordPress, and as such, the name of the function, or its functionality could change before the release. There is no guarantee this won't happen -  that's why we don't push features against non-stable code. 

So I'd very much prefer keeping this as-is, at least until the first RC (or the first beta at least). That will give us some time to react and update it.

How does that sound?

Reply:
That being said, I'm not even sure if I still have merge permissions, so I'll leave this to you folks to decide 😉